README.md

```diff
@@ -27,6 +27,7 @@ Please take a quick look at the [contribution guidelines](CONTRIBUTING.md) first
     - [Timeline](#timeline)
     - [Utils](#utils)
     - [Video Player](#video-player)
+    - [Loading](#loading)
```

Comment:
As described in the contribution.md at least 3 items are needed to create a new category. If you find at least two others you can easily create a new category